### DISCUSSION TITLE
I paid to use github copilot but it is not working<title>

### DISCUSSION THREAD

**Discussion by JoaoChrisostomo** [community/community#39540] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hello guys, I paid to use github copilot but it's not working, I don't know what the problem would be. who can i speak to from github support to resolve this.

![image](https://user-images.githubusercontent.com/83593479/202871006-7c0b366a-3cbe-496e-8c16-d7f06f16bd63.png)

I've already checked if I'm linked to the github account and it's also there in the corner of vsCode

**Comment by Yawan-1** [comment]
Is your console showing any errors in `output` section ?

**Reply by JoaoChrisostomo** [reply]
No problem is occurring in the output.
![image](https://user-images.githubusercontent.com/83593479/203467018-f3c91be2-87a3-4592-b81a-e9eb70cb7113.png)

**Comment by davecheney** [comment]
@JoaoChrisostomo 
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by JoaoChrisostomo** [reply]
What parts would these be that you are saying?

Where is the dropdown menu?

Below will be the image to see that I am logged in with my github.

![image](https://user-images.githubusercontent.com/83593479/203467176-2a05e3be-273d-472c-902b-56c09c737ac6.png)

**Reply by davecheney** [reply]
> Can you please paste the relevant parts of View -> Output; select GitHub Copilot from the pulldown.

**Reply by JoaoChrisostomo** [reply]
Do you have discord?

Could we exchange a chat there if you want to help me please solve this problem?

**Reply by davecheney** [reply]
I’m sorry I cannot provide assistance over discord. You can contact support via email if you prefer https://support.github.com/contact

**Reply by lheinbokel** [reply]
<img width="909" alt="image" src="https://user-images.githubusercontent.com/46933003/203869217-4fafd9fa-7317-4eff-8476-68cab0fa1c33.png">

I have the same issue. For me it's showing this error message @davecheney

**Comment by wolfgangmeyers** [comment]
I am getting the same error. I canceled my subscription, no way to resubscribe, no way to get this working again.

**Reply by lheinbokel** [reply]
I also canceled my subscription, until they get this fixed.

**Comment by alikhil** [comment]
I am agree. I have been using copilot with vscode during beta test and it seems like it have been working better and more stable than now. I don't feel that it helps me now

**Comment by DSDmark** [comment]
Hi @JoaoChrisostomo,

I'm not sure, but I think it's help you.
Open your settings and there are two sections you should check out. It may give you some info about your problem, I guess.

1. billing and payment.
2. copilot

---

![2022-11-25_12-27](https://user-images.githubusercontent.com/85744816/203915354-ec2d7350-e052-4742-b429-a9323d585d08.png)

And check this out may cause your problem:-(https://github.com/community/community/discussions/40022#discussioncomment-4234435)

In the last I think if you feel hopless then is only one last things you can do:- 
1. [**GitHub support Create Tickets**](https://support.github.com/request).
2. [**GitHub support**](https://support.github.com/)

**Reply by DSDmark** [reply]
So, I read a lot of discussion about similar issue like yours, and I think it's because of.

> This was caused by a bad deploy of copilot.

Don't worry, it will fix soon.

**Reply by wolfgangmeyers** [reply]
Yeah that would be great, it is ridiculous how crippled I felt when it stopped working hahaha

**Reply by lheinbokel** [reply]
@wolfgangmeyers Yes, exactly! all of the sudden I realized how helpful it really is!

**Reply by wolfgangmeyers** [reply]
yay it's working again. I guess I'll resubscribe again when it is possible to do so.

**Reply by lheinbokel** [reply]
> yay it's working again. I guess I'll resubscribe again when it is possible to do so.

Awesome, thanks for letting me know. Happy hacking!

**Comment by wolfgangmeyers** [comment]
Seeing this again
![image](https://user-images.githubusercontent.com/1783800/204064214-1bb13fc9-6a28-43e2-b6f1-65e804727811.png)
